src/Route/FastPsr7RouteInfo.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Penny\Route;
+
+class FastPsr7RouteInfo implements RouteInfoInterface
+{
+    private $name;
+    private $callable;
+    private $params;
+
+    public static function matched($name, callable $callable, $params = [])
```

Comment:
Yes :).. are you interested to write a PR? Otherwise I write a PR this evening :)